Src/Plugins/Security/SEC101_004.FacebookAppCredentialsValidator.cs

```diff
@@ -97,7 +105,12 @@ private ValidationState RetrieveInformation<T>(string url, string id, ref string
 
             try
             {
-                using HttpResponseMessage response = httpClient.GetAsync(url).GetAwaiter().GetResult();
```

Comment:
>httpClient.GetAsync(url) [](http://example.com/codeflow?start=53&length=24)

`DelegatingHandler` does not expose GetAsync. With that, we must change from GetAsync to SendAsync and pass the HttpRequestMessage.

This will enable our mocking framework.